packages/engines/toil/README.md

```diff
@@ -50,3 +50,41 @@ docker tag ${ECR_REPO}:latest ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com
 docker push ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com/${ECR_REPO}:latest
 ```
 
+### Building and Deploying on ARM
+
+If you are running on an ARM architecture system, `docker build` will try and build a Docker image for an ARM host. This will not work for two reasons. First, AGC uses AMD64 hosts and needs an AMD64 image. Second, the build will fail because the Erlang packages used by RabbitMQ are only available for AMD64. On ARM, the Erlang dependency will not be satisfiable:
```

Comment:
```suggestion
If you are running on an ARM architecture system, `docker build` will try and build a Docker image for an ARM host. This will not work for two reasons. First, Amazon Genomics CLI uses AMD64 hosts and needs an AMD64 image. Second, the build will fail because the Erlang packages used by RabbitMQ are only available for AMD64. On ARM, the Erlang dependency will not be satisfiable:
```

Reply:
Preference for full product name in docs when possible